docs/testing/releases/630.md

```diff
@@ -0,0 +1,228 @@
+
+1. Install [the Simple Custom CSS and JS plugin](https://wordpress.org/plugins/custom-css-js/).
+2. Add the following custom JS code and load it in the frontend in the footer:
+
+```
+const { __experimentalRegisterCheckoutFilters } = window.wc.blocksCheckout;
+__experimentalRegisterCheckoutFilters( 'my-hypothetical-deposit-plugin', {
+    cartItemClass: () => 'my-class',
+} );
+```
+
+3. Create a test page, add the cart block to it and save it.
+4. Go to the frontend and add a product to the cart.
+5. Go to the cart page and ensure that cart item has the CSS class `my-class`.
+6. Go to the checkout page and ensure that the cart item has the CSS class `my-class`.
+
+Example of a cart item with the CSS class `my-class`:
+
+![](https://user-images.githubusercontent.com/3323310/141903888-ecdf5884-77bc-4709-a4ac-bebe5dd962f1.png)
+
+### Fix incorrect text domain. ([5020](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5020))
+
+1. Set the site language to Spanish.
+2. Download the translation files via Dashboard → Updates.
+3. Create a test page, add the checkout block and save the page.
+4. Verify that all texts of the checkout block are available in Spanish in the editor.
+
+⚠️ Please note that some strings are not translated in the frontend. This problem will be addressed with ([5112](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5112)).
+
+### Fix ‘Country is required’ error on the Cart block when updating shipping address ([5129](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5129))
+
+1. Add a product to the cart and go to the cart page.
+2. Provide a shipping address in the shipping calculator and click the `Update` button.
+3. Provide a shipping address from a different country and click the `Update` button again.
+4. Verify that the new shipping address is visible.
+5. Verify that the error `Country is required` is not visible.
+6. Go to the checkout page and verify that the new shipping address is visible.
+7. Finalize the order and verify that the new shipping address is visible on the order details.
+
+### Fix button style of Mini Cart block for small screens. ([4907](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4907))
+
+1. Create a test page, add the columns block and save the page.
+2. Add the mini cart block to one of the columns.
+3. Verify that the mini card block contains the total, the number of items and the cart icon on a large screen (>= 768px).
+4. Verify that the mini card block only contains the number of items and the cart icon on a small screen (< 768px).
+
+Large screen:
+
+![](https://user-images.githubusercontent.com/3323310/141926396-82eb319f-502f-4c3b-9aba-c9a3be9fe4e3.png)
+
+Small screen:
+
+![](https://user-images.githubusercontent.com/3323310/141926415-dbcf8133-ea26-4e08-a806-aee7923f83d2.png)
```

Comment:
The Mini Cart block is behind a feature flag, so it shouldn't be included in the testing instructions yet. I added the `skip changelog` label to the relevant PR.

```suggestion
```